portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,26 +28,34 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private ArrayList<String> messages;
-
-  @Override
-  public void init() {
-    messages = new ArrayList<>();
-    messages.add("Bonjour, ca va?");
-    messages.add("Nda aga i di?");
-    messages.add("Hey, how are you?");
-  }
+  private ArrayList<String> comments = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Convert the arraylist to JSON
```

Comment:
nit: this comment isn't really necessary; see https://drive.google.com/file/d/1CxWFDHDR0VBk38EgySAdYDDVF2nQ73Nq/view